source/site-administrators/studio/bucket-storage-publishing.rst

```diff
@@ -0,0 +1,178 @@
+:is-up-to-date: True
+
+.. index:: Bucket Storage Publishing
+
+.. _bucket-storage-publishing:
+
+=========================
+Bucket Storage Publishing
+=========================
+
+Crafter CMS supports managing external assets in bucket storage (S3) including workflow and publishing mechanics.
+This allows uploading assets to a bucket storage (S3) for preview, that can then be published to a staging bucket storage, then finally to a live bucket storage, thus making the external assets available to delivery only after the assets have been published to the live bucket storage.
+
+-----------------------------------
+Publishing to AWS S3 Bucket Storage
+-----------------------------------
+
+Let's take a look at an example of setting up buckets (AWS S3) for preview, staging and live and then uploading then publishing assets to the buckets we setup.  Here are the steps:
+
+
+#. Enable staging
+#. Setup the blob store
+#. Create an ``s3`` folder under ``static-assets``
```

Comment:
@alhambrav this folder `/static-assets/s3` is just an example used in the config, I think it would be good to use a different example like `/static-assets/images/docs` or something to show that the feature can be included seamlessly in an existing site